lab-anthony/app/controllers/game-controller.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+//npm modules
+const angular = require('angular');
+
+//app modules
+const Monster = require('../models/monster.js');
+const Gem = require('../models/gem.js');
+
+//modules logic
+angular.module('monsterMash').controller('MonsterController', [MonsterController]);
+
+function MonsterController() {
```

Comment:
It seems weird to name it monstercontroller when the file is named gamecontroller.